lib/hooks/typography/index.ts

```diff
@@ -0,0 +1,56 @@
+import { useClassNames } from 'sku/treat';
+import { fontFamily, fontWeight, heading, text } from './typography.treat';
+import { useTheme } from '../../components/private/ThemeContext';
+
+export type TextWeight = 'regular' | 'medium' | 'strong';
+
+interface TextParams {
+  weight?: TextWeight;
+  size?: keyof typeof text;
+  baseline: boolean;
+}
+
+export const useText = ({
+  weight = 'regular',
+  size = 'standard',
+  baseline,
+}: TextParams) =>
+  useClassNames(fontFamily, fontWeight[weight], text[size].fontSize, {
+    [text[size].transform]: baseline,
+  });
+
+export type HeadingLevel = keyof typeof heading;
+export type HeadingWeight = 'regular' | 'weak';
+
+interface HeadingParams {
+  weight?: HeadingWeight;
+  level: HeadingLevel;
+  baseline: boolean;
+}
+
+const resolveLevelToken = (level: HeadingLevel) => {
+  const { tokens } = useTheme();
+
+  return {
+    '1': tokens.heading.level1,
```

Comment:
Should we just rename the heading tokens to `1`, `2` and `3` so we don't need to map?

Reply:
Considered that, it would make things simpler. Should we do this when we revisit the separation and structure of runtime tokens?

Reply:
Yep, sounds good 👍